docs/articles_en/learn_openvino/openvino_samples/model_creation.rst

```diff
@@ -80,8 +80,6 @@ To run the sample, you need to specify model weights and a device.
    - The ``lenet.bin`` weights file is generated by
      :doc:`model conversion API <openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide>`
      from the public LeNet model, with the ``input_shape [64,1,28,28]`` parameter specified.
-   - The original model is available in the
```

Comment:
This info is necessary in this context, we should not remove it... The whole sample is outdated, though and should be fixed.

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/22509/commits/ef18601b07d035c40555cb80978cca7d3fb76669